blog/2023/05-14-android-v250/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Jellyfin Android v2.5.0 - Bitrate selection, player improvements and quality-of-life tweaks'
```

Comment:
Title might be a bit too long, probably makes sense to the second part to the description